package.json

```diff
@@ -51,5 +54,8 @@
   },
   "scripts": {
     "test": "grunt ci --verbose"
+  },
+  "browser": {
+    "pouchdb/lib/deps/request": "pouchdb/lib/deps/request-browser"
```

Comment:
Agree, that seems pretty fragile. Shouldn't browserify automatically handle it for you, since you've got `pouchdb` in the `dependencies` and we've got [this line](https://github.com/pouchdb/pouchdb/blob/4d135c63efb3128a5e262eca345b451bf717a23e/package.json#L106)?